app.js

```diff
@@ -48,17 +49,20 @@ module.exports = () => {
     // Load the library data
     librariesUtil.set(app);
 
-    // Set all the relevant headers for the app
-    app.use((req, res, next) => {
-        res.header('Access-Control-Allow-Credentials', true);
-        res.header('Access-Control-Allow-Origin', '*');
-        res.header('Access-Control-Allow-Methods', 'GET,PUT,POST,DELETE,OPTIONS');
-        res.header('Access-Control-Allow-Headers', 'X-CSRF-Token, X-Requested-With, Accept, Accept-Version, Content-Length, Content-MD5, Content-Type, Date, X-Api-Version');
-        if (!localMode) {
+    // Set up cors headers
+    app.use(cors({
+        origin: '*',
+        credentials: true,
+        methods: ['GET', 'PUT', 'POST', 'DELETE', 'OPTIONS'],
+        allowedHeaders: ['X-CSRF-Token', ' X-Requested-With', ' Accept', ' Accept-Version', ' Content-Length', ' Content-MD5', ' Content-Type', ' Date', ' X-Api-Version'],
```

Comment:
There seems to be some extra spacing inside the strings here, is that intended?

Reply:
Oh my bad! I'll add tests for this.